mcs/class/corlib/corefx/SR.cs

```diff
@@ -999,4 +1000,46 @@ partial class SR
+	public const string ArgumentOutOfRange_OptionsInvalid = "options contains an invalid flag.";
+	public const string ArgumentOutOfRange_HandleInheritabilityNoneOrInheritable = "HandleInheritability.None or HandleInheritability.Inheritable required.";
+	public const string ArgumentOutOfRange_InvalidTimeout = "Timeout must be non-negative or equal to -1 (Timeout.Infinite)";
+	public const string ArgumentOutOfRange_MaxNumServerInstances = "maxNumberOfServerInstances must either be a value between 1 and 254, or NamedPipeServerStream.MaxAllowedServerInstances (to obtain the maximum number allowed by system resources).";
+	public const string InvalidOperation_PipeNotYetConnected = "Pipe hasn't been connected yet.";
+	public const string InvalidOperation_PipeDisconnected = "Pipe is in a disconnected state.";
+	public const string InvalidOperation_PipeHandleNotSet = "Pipe handle has not been set.  Did your PipeStream implementation call InitializeHandle?";
+	public const string InvalidOperation_PipeNotAsync = "Pipe is not opened in asynchronous mode.";
+	public const string InvalidOperation_PipeReadModeNotMessage = "ReadMode is not of PipeTransmissionMode.Message.";
+	public const string InvalidOperation_PipeAlreadyConnected = "Already in a connected state.";
+	public const string InvalidOperation_PipeAlreadyDisconnected = "Already in a disconnected state.";
+	public const string IO_IO_PipeBroken = "Pipe is broken.";
+	public const string IO_OperationAborted = "IO operation was aborted unexpectedly.";
+	public const string IO_PipeBroken = "Pipe is broken.";
+	public const string IO_InvalidPipeHandle = "Invalid pipe handle.";
+	public const string NotSupported_AnonymousPipeUnidirectional = "Anonymous pipes can only be in one direction.";
+	public const string NotSupported_AnonymousPipeMessagesNotSupported = "Anonymous pipes do not support PipeTransmissionMode.Message ReadMode.";
+	public const string ObjectDisposed_PipeClosed = "Cannot access a closed pipe.";
+	public const string PlatformNotSupported_MessageTransmissionMode = "Message transmission mode is not supported on this platform.";
+	public const string PlatformNotSupported_RemotePipes = "Access to remote named pipes is not supported on this platform.";
+	public const string PlatformNotSupported_InvalidPipeNameChars = "The name of a pipe on this platform must be a valid file name or a valid absolute path to a file name.";
+	public const string PlatformNotSupported_OperatingSystemError = "The operating system returned error '{0}' indicating that the operation is not supported.";
+	public const string IO_AllPipeInstancesAreBusy = "All pipe instances are busy.";
+	public const string UnauthorizedAccess_NotOwnedByCurrentUser = "Could not connect to the pipe because it was not owned by the current user.";
+	public const string UnauthorizedAccess_ClientIsNotCurrentUser = "Client connection (user id {0}) was refused because it was not owned by the current user (id {1}).";
```

Comment:
These changes look unnecessary

Reply:
I had to remove the reference to "referencesource" System.Core strings due to conflicts. Now all the strings come from CoreFX.

Reply:
sure but this is mscorlib change

Reply:
Ah, missed that. That was unintentional of course :-)